max-david/starter-code/new.html

```diff
@@ -0,0 +1,117 @@
+    <script id="article-template" type="text/x-handlebars-template">
+      <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+            By <address><a href="{{url}}">{{author}}</a></address>
+            {{publishStatus}}
+          </div>
+        </header>
+        <section class="article-body">{{{body}}}</section>
+        <a href="#" class="read-on">Read on &rarr;</a>
+      </article>
+    </script>
+    </head>
+  <body>
+    <header class="site-header clearfix">
+      <section class="heading-group-h1 pull-left">
+        <h1>
+          Kilovolt Blog!
+        </h1>
+        <h2>The shocking power of static pages</h2>
+      </section>
+      <nav class="main-nav">
+        <div class="icon-menu"></div>
+        <ul>
+          <!-- NOTE (stretch goal): In the same way that we enabled tab navigation on index.html, we can do the same on this page utilizing these elements. -->
+          <li class="tab" data-content="write"><a href="#" class="icon-home"> Write</a></li>
+          <li class="tab" data-content="articles"><a href="#" class="icon-home"> Preview</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="clearfix">
+
+      <section id="write" class="tab-content">
+        <h1>New Article</h1>
+        <!-- DONE: Set up a form to accept user input for a new blog article. Remember that we need:
+                   1. Article title
+                   2. Article body
+                   3. Article author
+                   4. Author's URL
+                   5. Article category
+                   6. Some way to know if this article is a draft or is published -->
+         <form class="" action="index.html" method="post">
+           <table>
+             <tr>
+               <td><input type="text" name="title" placeholder="Article Title"/></td>
+             </tr>
+             <tr>
+               <td><textarea name="body" placeholder="Article Body"></textarea></td>
+             </tr>
+             <tr>
+               <td><input type="text" name="author" placeholder="Article Author"/></td>
+             </tr>
+             <tr>
+               <td><input type="text" name="authorUrl" placeholder="Article URL"/></td>
+             </tr>
+             <tr>
+               <td><input type="text" name="category" placeholder="Article Category"/></td>
+             </tr>
+             <tr>
+               <td>
+                 <input class="check" type="checkbox" name="publishStatus" />
+                 <label for="publishStatus">Published</label>
+               </td>
+             </tr>
+           </table>
+         </form>
+      </section>
+
+      <section id="articles" class="tab-content">
+        <!-- NOTE: This is where we will display our article preview -->
+        <pre><code class="precode"></code></pre>
```

Comment:
You do not need to have this line here. Highlight.js is being used on the body of the article when it is added to the preview. This means that if the user writes in the body `textarea`
```
<pre><code class="precode">
var foo = 'bar';
</code></pre>
```
it will be highlighted like actual code :)